src/pages/gateway/getting-started.md

```diff
@@ -12,7 +12,7 @@ This guide provides you with the basic steps you need to set up your API managem
 Before you can begin using the API Mesh, you must install:
 
 -  [Node.js]
--  [nvm] 16.x.x
+-  [nvm] 16.x.x (Mac/Linux) or [nvm-windows](https://github.com/coreybutler/nvm-windows)(Windows)
 
```

Comment:
Also please change `schema mangement plugin` to `api-mesh plugin` on line 26 below to maintain consistency